cmd/jaeger/internal/integration/e2e_integration.go

```diff
@@ -215,6 +215,9 @@ func createStorageCleanerConfig(t *testing.T, configFile string, storage string)
 
 func purge(t *testing.T) {
 	addr := fmt.Sprintf("http://0.0.0.0:%s%s", storagecleaner.Port, storagecleaner.URL)
+	if purgerAddr, ok := os.LookupEnv("PURGER_ADDRESS"); ok {
```

Comment:
I think it's better to call this ENDPOINT, since it's not just host:port address.